source/components/atoms/Stepper/Stepper.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
Do we really need a story for this component . Because this one doesn't have a design to view. . Its more logical component :)

Reply:
Yeah, i tend to add all the components in stories. I think it's good to get a good overview, i see it as documentation for the components that we have. However the documentation is poor at the moment :P